servers/src/main/java/tachyon/worker/file/FileSystemMasterClient.java

```diff
@@ -101,4 +103,23 @@ public FileInfo call() throws TException {
       }
     });
   }
+
+  /**
+   * Instructs a worker to persist the files.
```

Comment:
Yeah, right now the only command is PersistCommand, but I can imagine the heartbeat can carry other commands in the future. Fixed.